delegation-verify-coordinator/templates/deployment.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "delegation-verify-coordinator.fullname" . }}
+  labels:
+    {{- include "delegation-verify-coordinator.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "delegation-verify-coordinator.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "delegation-verify-coordinator.labels" . | nindent 8 }}
+	      {{- with .Values.podLabels }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      serviceAccountName: {{ include "delegation-verify-coordinator.serviceAccountName" . }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            # TODO values
+            {{- with .Values.env }}
+            - name: SURVEY_INTERVAL_MINUTES
+              value: {{ .surveyIntervalMinutes }}
+            - name: POSTGRES_HOST
+              value: {{ .postgres.host }}
+            - name: POSTGRES_PORT
+              value: {{ .postgres.port }}
+            - name: POSTGRES_DB
+              value: {{ .postgres.db }}
+            - name: POSTGRES_USER
+              value: {{ .postgres.user }}
+            - name: POSTGRES_PASSWORD
+              value: {{ .postgres.password }}
+            - name: MINI_BATCH_NUMBER
+              value: {{ .miniBatchNumber }}
+            - name: UPTIME_DAYS_FOR_SCORE
+              value: {{ .uptimeDaysForScore }}
+            - name: WORKER_TAG
+              value: {{ .workerTag }}
+            - name: WORKER_IMAGE
+              value: {{ .workerImage }}
+            - name: AWS_KEYSPACE
+              value: {{ .aws.keyspace }}
+            - name: AWS_REGION
+              value: {{ .aws.region }}
+            - name: AWS_ACCESS_KEY_ID
+              value: {{ .aws.accessKeyID }}
+            - name: AWS_SECRET_ACCESS_KEY
+              value: {{ .aws.secretAccessKey }}
+            - name: AWS_SSL_CERTIFICATE_PATH
+              value: {{ .aws.sslCertificatePath }}
```

Comment:
We discussed that instead of getting to cassandra through aws authentication, we probably in the future will want to push for talking to keyspace instances exposed through alb. Then your application would not care if you would connect to cassandra managed by aws or installed locally on your docker.